cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/release.go

```diff
@@ -185,3 +259,44 @@ func installedSummaryStatusFromUnstructured(unstructuredRelease map[string]inter
 	}
 	return status
 }
+
+func installedPackageReconciliationOptionsFromUnstructured(unstructuredRelease map[string]interface{}) *corev1.ReconciliationOptions {
+	reconciliationOptions := &corev1.ReconciliationOptions{}
+	if intervalString, found, err := unstructured.NestedString(unstructuredRelease, "spec", "interval"); found && err == nil {
+		if duration, err := time.ParseDuration(intervalString); err == nil {
+			reconciliationOptions.Interval = int32(duration.Seconds())
+		}
+	}
+	if suspend, found, err := unstructured.NestedBool(unstructuredRelease, "spec", "suspend"); found && err == nil {
+		reconciliationOptions.Suspend = suspend
+	}
+	if serviceAccountName, found, err := unstructured.NestedString(unstructuredRelease, "spec", "serviceAccountName"); found && err == nil {
+		reconciliationOptions.ServiceAccountName = serviceAccountName
+	}
+	return reconciliationOptions
+}
+
+func installedPackageAvailablePackageRefFromUnstructured(unstructuredRelease map[string]interface{}) (*corev1.AvailablePackageReference, error) {
+	repoName, found, err := unstructured.NestedString(unstructuredRelease, "spec", "chart", "spec", "sourceRef", "name")
+	if !found || err != nil {
+		return nil, status.Errorf(codes.Internal, "missing required field spec.chart.spec.sourceRef.name")
+	}
+	chartName, found, err := unstructured.NestedString(unstructuredRelease, "spec", "chart", "spec", "chart")
+	if !found || err != nil {
+		return nil, status.Errorf(codes.Internal, "missing required field spec.chart.spec.chart")
+	}
+	repoNamespace, _, _ := unstructured.NestedString(unstructuredRelease, "spec", "chart", "spec", "sourceRef", "namespace")
```

Comment:
Just wondering why the err is ignored here?

Reply:
my bad, will fix. Thanks for reviewing